qiskit/transpiler/passes/layout/vf2_post_layout.py

```diff
@@ -66,15 +67,16 @@ class VF2PostLayout(AnalysisPass):
 
     If a solution is found that means there is a lower error layout available for the
     circuit. If a solution is found the layout will be set in the property set as
-    ``property_set['post_layout']``. However, if no solution is found, no
+    ``property_set['post_layout']``. However, if no solution or no better solution is found, no
     ``property_set['post_layout']`` is set. The stopping reason is
```

Comment:
This isn't quite true - if no better solution is found, we're setting this now.

Reply:
Thanks for your input! We are not setting ``property_set['post_layout']`` in that commit, if "no better solution found" applies. If I understood Matthew correctly, setting the post_layout field to None when it equals the initial layout of a quantum circuit helps to avoid the execution of extra steps -- so setting post_layout in that case is detrimental to the runtime of the transpiler...

Reply:
Oh sorry my bad, I read the bottom bit wrong.